recipes/ctng-compilers/meta.yaml

```
@@ -0,0 +1,258 @@
+        - {{ pin_subpackage("gcc_impl_" ~ ctng_target_platform, exact=True) }}
+    run_exports:
+      # impose this requirement across the build/host boundary
+      strong:
+        - {{ pin_subpackage("libstdcxx-ng", max_pin=None) }}
+    test:
+      files:
+        - tests
+      commands:
+        - CXX=$(${PREFIX}/bin/*-gcc -dumpmachine)-g++
+        - ${CXX} -Wall tests/aligned_alloc.cpp -o cpp_aligned --std=c++17
+        - ./cpp_aligned
+    about:
+      summary: GNU C++ Compiler
+      home: https://gcc.gnu.org/
+      license: GPL
+
+  - name: gfortran_impl_{{ ctng_target_platform }}
+    script: install-gfortran.sh
+    build:
+      number: {{ build_num }}
+      detect_binary_files_with_prefix: False
+      binary_relocation:
+        - 'bin/*'
+        - 'lib/*.so'
+        - 'libexec/**/*.so*'
+        - 'libexec/**/f951'
+      missing_dso_whitelist:
+        - "*"
+    requirements:
+      build:
+        # For cpp and crt{i,n}.o
+        - {{ pin_subpackage("gcc_impl_" ~ ctng_target_platform, exact=True) }}
+      run:
+        # For cpp and crt{i,n}.o
+        - gcc_impl_{{ ctng_target_platform }} >=7.2.0
+        - {{ pin_subpackage("libgfortran-ng") }}
+        - {{ pin_subpackage("libgcc-ng", max_pin=None) }}
+    run_exports:
+      # impose this requirement across the build/host boundary
+      strong:
+        - {{ pin_subpackage("libgfortran-ng") }}
+        - {{ pin_subpackage("libgcc-ng", max_pin=None) }}
+    test:
+      requires:
+        - cmake >=3.11
+      commands:
+        - find $PREFIX/lib -iname omp_lib.mod | grep '.'
+        - find $PREFIX/lib -iname omp_lib.h | grep '.'
+        - pushd tests/fortomp
+        - sh test_fort.sh
+      files:
+        - tests/fortomp/*
+    about:
+      summary: GNU Fortran Compiler
+      home: https://gcc.gnu.org/
+      license: GPL
+
+  - name: binutils_impl_{{ ctng_target_platform }}
+    version: "{{ ctng_binutils }}"
+    script: install-binutils.sh
+    build:
+      number: {{ build_num }}
+      detect_binary_files_with_prefix: False
+      binary_relocation:
+        - 'bin/*'
+        - 'lib/*.so'
+        - 'libexec/**/*.so*'
+      missing_dso_whitelist:
+        - "*"
+    test:
+      commands:
+        - test -f ${PREFIX}/{{ ctng_cpu_arch }}-*/bin/ar
+        - test -f ${PREFIX}/{{ ctng_cpu_arch }}-*/bin/ld
+    about:
+      summary: The GNU Binutils are a collection of binary tools.
+      home: https://www.gnu.org/software/binutils/
+      license: GPL
+
+  - name: libstdcxx-ng
+    target: {{ ctng_target_platform }}
+    script: install-libstdc++.sh
+    build:
+      number: {{ build_num }}
+      detect_binary_files_with_prefix: False
+      binary_relocation: False
+      missing_dso_whitelist:
+        - "*"
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libstdc++.so
+    about:
+      summary: The GNU C++ Runtime Library
+      home: https://gcc.gnu.org/
+      license: GPL3 with runtime exception
```

The SPDX identifier is:
```suggestion
      license: GPL-3.0-with-GCC-exception
      license_family: GPL
```
Should there also be `license_file`?
(Also applies to the other outputs)

---

licenses are packaged in share, so there's no need to add them explicitly.